pages/your_domain/microbial_biotechnology.md

```diff
@@ -7,151 +7,227 @@ related_pages:
 * Enzymes can be assigned EC numbers, according to the guidance from the [International Union of Biochemistry and Molecular Biology (IUBMB)](https://www.qmul.ac.uk/sbcs/iubmb/), to indicate their function and an entry made in the [BRaunschweig ENzyme DAtabase](https://www.brenda-enzymes.org/) (BRENDA).
 * More generally, the [IUPAC-IUBMB Joint Commission on Biochemical Nomenclature (JCBN)](https://www.qmul.ac.uk/sbcs/iupac/jcbn/) encourages the communication of biochemical information using generally understood terminology.
 
-### Model based designs
+#### (Meta)Data publication
+* SBOL.
+* Databases such as SBOLME ([Kuwahara et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28076956/)) or [SynBioHub](https://synbiohub.org) can be used to share the data.
+* Metabolite information can also be submitted to, or referred to in, [ChEBI](https://www.ebi.ac.uk/chebi/).
+* [BRaunschweig ENzyme DAtabase](https://www.brenda-enzymes.org/) (BRENDA).
+
+
+## Design: Mathematical model - Standards and (meta)data publication
+
+### Description
+What tools and standards need to be considered when building mathematical models to aid the design of genetic systems?
 
-#### Considerations
-* What tools and standards need to be considered when building mathematical models to aid the design of genetic systems
-* How can the models be shared via repositories and made  available in a way that makes results replicable?
+How can the models be shared via repositories and made  available in a way that makes results replicable?
 
-#### Solutions
+### Considerations
+* A variety of standards and tools are available for model building.
+* It is important to associate the genetic design with its corresponding model.
 
+### Solutions
 * [Systems Biology Markup Language (SBML)](http://sbml.org/Main_Page) is a popular standardised format for sharing mathematical models for which a variety of tools are available for model building.
-* Models can be shared in model repositories such as [biomodels](https://www.ebi.ac.uk/biomodels/).
-* SBOL can also be used to associate a genetic design with its corresponding model. 
 * More generally, the [COmputational Modeling in BIology NEtwork (COMBINE)](http://co.mbine.org/), provides a platform for coordinating standardisation of models in biology.
+* SBOL can also be used to associate a genetic design with its corresponding model.
+* Models can be shared in model repositories such as [biomodels](https://www.ebi.ac.uk/biomodels/).
 
 
-## Build 
+## Build: Methods - Documentation and (meta)data publication
 
-### Description: 
+### Description
+The build stage in the microbial biotechnology and/or synthetic biology life cycle involves the application of any number of a range of experimental techniques and, since these techniques are so varied, the domain is therefore very difficult to standardise in terms of the data and metadata to be shared. 
```

Comment:
There is repeated information here from the overall introduction. This is fine if it is intentional as it reminds the reader of the build stage.

Reply:
I think it is intentional indeed.